spec/models/user_spec.rb

```diff
@@ -586,10 +586,6 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
     expect(user.decorate.config_body_class).to eq("default default-article-body pro-status-#{user.pro?} trusted-status-#{user.trusted}")
   end
 
-  it "inserts into mailchimp" do
```

Comment:
I suggest adding test(s) for the `subscribe_to_mailchimp_newsletter` instead of this, like test that it schedules a job when the conditions are met.